doc/classes/EditorSettings.xml

```diff
@@ -961,7 +961,17 @@
 			If [code]true[/code], on Linux/BSD, the editor will check for Wayland first instead of X11 (if available).
 		</member>
 		<member name="run/window_placement/android_window" type="int" setter="" getter="">
-			The Android window to display the project on when starting the project from the editor.
+			Specifies how the Play window is launched relative to the Android editor.
+			- [b]Auto (based on screen size)[/b] (default) will automatically choose how to launch the Play window based on the device and screen metrics. Defaults to [b]Same as Editor[/b] on phones and [b]Side-by-side with Editor[/b] on tablets.
+			- [b]Same as Editor[/b] will launch the Play window in the same window as the Editor.
+			- [b]Side-by-side with Editor[/b] will launch the Play window side-by-side with the Editor window.
+			[b]Note:[/b] Only available in the Android editor.
+		</member>
+		<member name="run/window_placement/play_window_pip_mode" type="int" setter="" getter="">
+			Specifies the picture-in-picture (PiP) mode for the Play window.
+			- [b]Disabled[/b] (default for non-Android editor) PiP is disabled for the Play window.
+			- [b]Enabled[/b] If the device supports it, PiP is always enabled for the Play window. The Play window will contain a button to enter PiP mode.
+			- [b]Enabled when Play window is same as Editor[/b] (default for Android editor) If the device supports it, PiP is enabled when the Play window is the same as the Editor. The Play window will contain a button to enter PiP mode.
```

Comment:
```suggestion
			- [b]Disabled[/b] (default for non-Android editor): PiP is disabled for the Play window.
			- [b]Enabled:[/b] If the device supports it, PiP is always enabled for the Play window. The Play window will contain a button to enter PiP mode.
			- [b]Enabled when Play window is same as Editor[/b] (default for Android editor): If the device supports it, PiP is enabled when the Play window is the same as the Editor. The Play window will contain a button to enter PiP mode.
```
That being said:

> [b]Disabled[/b] (default for non-Android editor): PiP is disabled for the Play window.

seems to contradict:

> [b]Note:[/b] Only available in the Android editor.

So maybe that parenthesis should be removed.

Reply:
Done!